transformato/tests/test_transformato.py

```diff
@@ -53,6 +53,7 @@ def test_transformato_imported():
 
 
 def test_read_yaml():
+    """Sample test, will check ability to read yaml files"""
```

Comment:
very nice!
I have started using this:
https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring
to generate consistent doc strings.
But for a test a one-liner is perfectly fine as comment.